### PR TITLE
Ocd 2781 - Improve Reports responsiveness - page load - Product/Version Activity

### PR DESCRIPTION
**Pull request by tmy1313**
Contains commits related to OCD-2755

### CODE REVIEW

File: PENDING_RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
+* Create table 'filter_type' to indicate the type of filter
+
 ### View Modifications
 * Updated "acb_is_deleted" field in certified_product_details view to be "acb_is_retired".
 * Created view "listings_from_banned_developers" to query when constructing the Banned Developers API response.
 
----
\ No newline at end of file
+### Data Modifications
+* Add listng, developer, product, and version reports to the filter_type table
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]:
```suggestion
* Add listing, developer, product, and version reports to the filter_type table
```

**Comment by kekey1** [comment]
Approved but needs release notes merged.

**Comment by andlar** [comment]
The thing that's left is removing the old Reports. https://github.com/chpladmin/chpl-website/pull/469/commits/bd104c058f7fa8dea349a444af0e3cd64c2858cf is the commit where I moved the ACB report to the new page. It looks a lot like this PR, but this PR doesn't have all of the "removals" from the admin/reports/reports js & html files

File: PENDING_RELEASE_NOTES.md [hunk 2]
```diff
@@ -3,10 +3,16 @@
 _Date TBD_
 
 ### View/Table modifications
+>>>>>>> development
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]: Merge issue

File: PENDING_RELEASE_NOTES.md [hunk 3]
```diff
@@ -3,10 +3,16 @@
...
 * Create table 'filter' to handle storing user's filters
-* Create table 'filter_type' to indicate the type of filter 
+* Create table 'filter_type' to indicate the type of filter
+
+### View Modifications
```
Comment (andlar) [inline PENDING_RELEASE_NOTES.md]: Not sure why we have "View/Table modifications" separate from "View" modifications. Should either be combined or split

File: PENDING_RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,11 +2,16 @@
...
 _Date TBD_
 
-### View/Table modifications
+### Table modifications
 * Create table 'filter' to handle storing user's filters
-* Create table 'filter_type' to indicate the type of filter 
+* Create table 'filter_type' to indicate the type of filter
+
+### View Modifications
+* Updated "acb_is_deleted" field in certified_product_details view to be "acb_is_retired".
```
Comment (kekey1) [inline PENDING_RELEASE_NOTES.md]: Although git will let you merge this file, this line and the one below were already deployed in the previous release. Check existing release notes when merging and update accordingly.

**Pull request merged**